src/main/java/ch/njol/skript/test/runner/ExprTestOfflinePlayer.java

```diff
@@ -0,0 +1,52 @@
+package ch.njol.skript.test.runner;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.NoDoc;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@NoDoc
+public class ExprTestOfflinePlayer extends SimpleExpression<OfflinePlayer> {
+
+	private static final OfflinePlayer PLAYER = Bukkit.getOfflinePlayer("Notch");
```

Comment:
Can we use the UUID method instead to avoid reliance on the Mojang servers for tests?
Ideally we make a whole profile ourselves to have full functionality without any webrequests but I think that's only possible on Paper.
(I'm also not a huge fan of using Notch specifically tbh)